concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -146,8 +146,8 @@ Hale-1973-1798-144	Human_Body-45	buttock	n	02a.45	834	BUTTOCKS
 Hale-1973-1798-145	Human_Body-46	rectum	n	02a.46	1935	ANUS
 Hale-1973-1798-146	Human_Body-47	arm	n	02a.47	1673	ARM
 Hale-1973-1798-147	Human_Body-48	shoulder blade	n	02a.48	172	SHOULDERBLADE
-Hale-1973-1798-148	Human_Body-49	right arm	n	02a.49	2183	RIGHT HAND
-Hale-1973-1798-149	Human_Body-50	left arm	n	02a.50	2182	LEFT HAND
+Hale-1973-1798-148	Human_Body-49	right arm	n	02a.49		
```

Comment:
checked this with the source, right now, you're right, this is not "left hand", or we can't confirm from the source.

Reply:
I checked with the actual data, and many forms for right/left arm contain the word for arm in the respective language. Hand is not even a concept (only parts of the hand). In some it contains the words for hand, but in almost all these cases hand and arm colexify fully. So, arm is what is meant.

Reply:
How did you check the data? I am asking, because the data we receive in our lexibank is extremely sparse, you find only one or two languages with the concepts...

Reply:
https://www.sil.org/system/files/reapdata/11/35/05/113505358959275586823513225753342608301/26456.pdf